include/MiniLua/string.hpp

```diff
@@ -0,0 +1,25 @@
+#ifndef MINILUA_STRING
+#define MINILUA_STRING
+#endif
+
+#include <MiniLua/values.hpp>
+
+namespace minilua::string {
+auto byte(const CallContext& ctx) -> Vallist;
+auto Char(const CallContext& ctx) -> Value;
```

Comment:
calling the function Char because char is reserved is a bad workaround imo because it is the only function starting with an uppercase letter. Possible solutions: _char, builtin_char, string_char, lua_char or something similar